src/pgen/cluster/agn_triggering.cpp

```diff
@@ -437,20 +466,34 @@ AGNTriggeringReduceTriggering(parthenon::MeshData<parthenon::Real> *md,
     break;
   }
   }
+  if (agn_triggering.triggering_mode_ == AGNTriggeringMode::COLD_GAS) {
+    auto &agn_triggering = hydro_pkg->Param<AGNTriggering>("agn_triggering"); // Make it non-const
+    const parthenon::Real cold_mass = hydro_pkg->Param<Real>("agn_triggering_cold_mass");
+    const parthenon::Real total_mass = hydro_pkg->Param<Real>("agn_triggering_total_mass");
+
+    hydro_pkg->UpdateParam<Real>("agn_triggering_prev_cold_mass", cold_mass); //Move from AGNTriggeringFinalizeTriggering
+    hydro_pkg->UpdateParam<Real>("agn_triggering_prev_total_mass", total_mass);
+  }
+  hydro_pkg->UpdateParam<AGNTriggering>("agn_triggering", agn_triggering);
```

Comment:
What's the intent here?
As far as I can tell you're already updating the `prev` values above.